README.md

```diff
@@ -5,103 +5,111 @@
 
-## Development Environment
-
-To develop this extension, the following environment is used:
+---
+## Working on Augury
 
-* Node 4.2.3
-* NPM 3.5.3
-* TypeScript 1.8.9
-* typings 1.3.2
+### Development environment
 
-## Installation
+To develop the Augury extension, the following environment is used:
 
-### Pre-packaged
-
-- You can install this extension from the [Chrome Webstore](https://chrome.google.com/webstore/detail/augury/elgalmkoelokbchhkhacckoklkejnhcd?hl=en-US)
+* Node
+* NPM
+* TypeScript
 
 ### Building & installing locally
 
-1. Clone this repository: `git clone git://github.com/rangle/augury`.
-2. Run `npm install`.
-3. Run `npm run dev-build`.
-4. Navigate to chrome://extensions and enable Developer Mode.
-5. Choose "Load unpacked extension".
-6. In the dialog, open the directory you just cloned.
+```bash
+git clone git://github.com/rangle/augury
+cd augury
+npm install
+npm run dev-build
+```
+
+1. Navigate to chrome://extensions and enable Developer mode.
+1. Choose "Load unpacked extension".
+1. In the dialog, open the directory you just cloned.
 
-Try out the extension with one of our [example applications](#examples).
+Try out the extension with one of the example app from the [Guide](https://augury.angular.io/pages/guides/).
 
-## Running Tests
+### Running tests
 
 To execute all unit tests, run `npm test`. It bundles up all files that match `*.test.ts` into `build/test.js`, then runs it through tape-run in a headless Electron browser.
 
-## Available NPM Scripts
+### Available NPM scripts
 
-- `build` Build the extension
-- `webpack` Run webpack
-- `clean` Clean `node_modules` and `typings`,
-- `postinstall` install typings
-- `start` Clean build and run webpack in watch mode
-- `test` Bundle all *.test.ts and run it through a headless browser
-- `prepack` Run npm build before running npm pack
-- `pack` Packages the extension and create chrome build augury.crx
+To see all available script type `npm run` in the terminal. The following command are the ones you will mostly be working with.
 
-## Examples
+Command|Descrption
+-------|----------
+`build`|Build the extension
+`webpack`|Run webpack
+`clean`|Clean `node_modules` and `typings`,
+`postinstall`|install typings
+`start`|Clean build and run webpack in watch mode
+`test`|Bundle all *.test.ts and run it through a headless browser
+`prepack`|Run npm build before running npm pack
+`pack`|Packages the extension and create chrome build augury.crx
 
-- [Kitchen Sink Demo](./example-apps/kitchen-sink-example/README.md)
-- [Superhero Tinder](./example-apps/superhero-app/README.md)
+## Reporting issues
 
-## Issues
+Please search to make sure your issue is not already been reported.
 
-### Known Issues
+You should report an issue directly from Augury, by clicking on the Augury icon next to the address bar in the browser. It will open up a popup menu with a link to Issue reporting.
```

Comment:
That's mentioned in the generated PR, there are basics pre-populated steps for them to follow. I didn't mentioned code snipped, so I will work that in.